api/provider/aws/common.go

```diff
@@ -278,3 +279,32 @@ func readService(ecsapi ecsiface.ECSAPI, clusterName, serviceID string) (*ecs.Se
 
 	return output.Services[0], nil
 }
+
+func waitTillSGDeletedFN(ec2api ec2iface.EC2API, securityGroupName string) func() (shouldRetry bool, err error) {
```

Comment:
is there a reason we're using "Till" instead of "Until" for this function name?

Reply:
Nope, I can fix that